services/r2ctl/config/r2ctl.go

```diff
@@ -195,3 +199,11 @@ type policiesConfiguration struct {
 	StoragePolicies  []policy.StoragePolicy   `yaml:"storagePolicies"`
 	AggregationTypes []policy.AggregationType `yaml:"aggregationTypes"`
 }
+
+func decodeRuneInString(s string) []rune {
+	var r []rune
```

Comment:
nit: preallocate enough so we don't need to reallocate, i.e., `r := make([]rune, 0, len(s))`. It might over-allocate a little in case of multi-byte runes but should work in most cases.